app.coffee

```diff
@@ -45,7 +45,7 @@ io.configure ->
 	# gzip uses a Node 0.8.x method of calling the gzip program which
 	# doesn't work with 0.6.x
 	#io.enable('browser client gzip')
-	io.set('transports', ['websocket', 'flashsocket', 'htmlfile', 'xhr-polling', 'jsonp-polling'])
```

Comment:
I think it is fine to loose `flashsocket`, @briangough I don't think we see much of them, agree?

Reply:
Yes we should remove that, we don't need to support flash these days.